packaging/ubuntu-trusty/ecs.conf

```diff
@@ -0,0 +1,21 @@
+# Copyright 2014-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the
+# "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and
+# limitations under the License.
+
+description "Amazon EC2 Container Service init"
+author "Amazon Web Services"
+start on (started docker and stopped cloud-final)
+
```

Comment:
Could be.  So far we've left it out of the Upstart file that we use on the Amazon Linux AMI since ecs-init itself should be restarting the agent.  Getting the [respawn policy](http://upstart.ubuntu.com/cookbook/#respawn) right is a bit tricky and I want to make sure that we get it exactly right before I put it in.